src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar
+
+data class Cars(val cars: List<Car>) {
+
+    fun move(moveStrategy: MoveStrategy): Cars {
+        val newCars = mutableListOf<Car>()
+        cars.forEach { newCars.add(it.move(moveStrategy.isMove())) }
+        return Cars(newCars)
+    }
```

Comment:
```suggestion
    fun move(moveStrategy: MoveStrategy): Cars {
        val newCars = cars.map { it.move(moveStrategy.isMove()) }
        return Cars(newCars)
    }
```

java에서는 `Stream API`의 `map`을 사용해야 가능하지만, 코틀린에는 `CollectionAPI`에 기본적으로 `map`이 있습니다.

`map`을 사용해서 새로운 리스트를 만들어주는 것도 좋아보이네요 😊